entrypoint.sh

```diff
@@ -26,7 +29,7 @@ echo "End: Sonar Scan"
 echo "Container Push: $INPUT_CONTAINER_PUSH_ENABLED"
 if [ "$INPUT_CONTAINER_PUSH_ENABLED" = 'true' ]; then
   echo "Start: Publish Image to ECR"
-  /scripts/publish.sh
+  ./actions-collection/scripts/publish.sh
   echo "End: Publish Image to ECR"
 fi
 
```

Comment:
Do `sudo git clean -fdx` at the end to clean up all the modifications

Reply:
Added.